### PR TITLE
[hi] Localize content/en/kubernetes.md into Hi

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Localize kubernetes.md
### Checklist before opening this PR (put `x` in the checkboxes)
- [ ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/hi/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: कुबेरनेट्स
+status: पूरा हुआ
+category: तकनीकी
+tags: ["infrastructure", "", ""]
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक खुला स्रोत कंटेनर ऑर्केस्ट्रेटर है जो
```
Comment (sayantani11) [inline content/hi/kubernetes.md:10]: Replace `खुला स्रोत` with `open source` but written in hindi

File: content/hi/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+आधुनिक अवसंरचना पर [कंटेनराइझड़](/container/) अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है।
+यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित प्रणाली] (/distributed-systems/) पर चल रहे अनुप्रयोगों का प्रबंधन करता है
+(ठीक आपके लैपटॉप के OS की तरह जो आपके ऐप्स को प्रबंधित करता है)।
```
Comment (sayantani11) [inline content/hi/kubernetes.md:13]: replace with `ऐप्स को प्रबंध करता है`

File: content/hi/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक खुला स्रोत कंटेनर ऑर्केस्ट्रेटर है जो
+आधुनिक अवसंरचना पर [कंटेनराइझड़](/container/) अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है।
+यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित प्रणाली] (/distributed-systems/) पर चल रहे अनुप्रयोगों का प्रबंधन करता है
+(ठीक आपके लैपटॉप के OS की तरह जो आपके ऐप्स को प्रबंधित करता है)। 
+
+कुबेरनेट्स एक [क्लस्टर] (/cluster/) में [नोड्स] (/nodes/) में कंटेनरों को शेड्यूल करता है।
```
Comment (sayantani11) [inline content/hi/kubernetes.md:15]: nit: कुबेरनेट्स एक [क्लस्टर] (/cluster/) के [नोड्स] (/nodes/) में कंटेनरों

File: content/hi/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक खुला स्रोत कंटेनर ऑर्केस्ट्रेटर है जो
+आधुनिक अवसंरचना पर [कंटेनराइझड़](/container/) अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है।
+यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित प्रणाली] (/distributed-systems/) पर चल रहे अनुप्रयोगों का प्रबंधन करता है
+(ठीक आपके लैपटॉप के OS की तरह जो आपके ऐप्स को प्रबंधित करता है)। 
+
+कुबेरनेट्स एक [क्लस्टर] (/cluster/) में [नोड्स] (/nodes/) में कंटेनरों को शेड्यूल करता है। 
+यह कई बुनियादी ढांचे के निर्माण को बंडल करता है, जिसे कभी-कभी "आदिम" कहा जाता है।
```
Comment (sayantani11) [inline content/hi/kubernetes.md:16]: I would like to know from other reviewers if we should write `"आदिम"` or primitive but written in Hindi.
@anubha-v-ardhan @jayesh-srivastava @Garima-Negi

File: content/hi/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+एक ऐप के उदाहरण की तरह, बैलेंसर्स लोड करें, लगातार स्टोरेज, और अन्य एक साथ
+इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है।
```
Comment (sayantani11) [inline content/hi/kubernetes.md:18]: `बैलेंसर्स लोड करें, लगातार स्टोरेज, और अन्य एक साथ इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है। `
load balancers, persistent storage are technical terms so we localise it not translate it.

Comment (sayantani11) [inline content/hi/kubernetes.md:18]: `अनुप्रयोगों` here should be replaced by apps/applications

Comment (sayantani11) [inline content/hi/kubernetes.md:18]: `अनुप्रयोगों` here should be replaced by apps/applications

**Review by sayantani11 — CHANGES_REQUESTED** [review]
I have reviewed as far as I read. I will take a look at the rest soon.

File: content/hi/kubernetes.md (hunk 3 above)
Comment (amolmote) [inline content/hi/kubernetes.md:15]:
```suggestion
कुबेरनेट्स एक [क्लस्टर] (/cluster/) के [नोड्स] (/nodes/) में कंटेनरों को शेड्यूल करता है। 
```

**Comment by amolmote** [comment]
its my bad @sayantani11, someone already opened PR of this. Thank you for your suggestions.

**Pull request closed without merging**